implementations/rust/ockam/ockam_app/src/enroll/enroll_user.rs

```diff
@@ -27,8 +27,7 @@ pub async fn enroll_user(app: &AppHandle<Wry>) -> Result<()> {
     let app_state: State<AppState> = app.state::<AppState>();
     enroll_with_token(app, &app_state)
         .await
-        .map(|i| info!("Enrolled a new user with identifier {}", i))
```

Comment:
I would still leave the logs on if we ever need to diagnose what happened for a specific user.

Reply:
I just moved that into the `enroll_with_token` function: https://github.com/build-trust/ockam/pull/5753/files#diff-24be761feb7cea4a0282d3086c1eeeaa89a3655cbbad8f7cc41abec52a785cf9R82

Reply:
Ah, great I did not notice this, thanks.